From 71d45b2e979a8844eddfe796afdb542ece7262ed Mon Sep 17 00:00:00 2001
From: yxj <yxj@rock-chips.com>
Date: Tue, 24 Jul 2012 11:38:40 +0800
Subject: [PATCH] rkfb: fix bug in commit
 7d80392f6c0338239b7d1e9b7c80a2343e23f118

      rkfb and lcdc driver use different macro to distinguish data format
      rkfb use macro defined by android,lcdc use marco to better match the
      lcdc register
---
 drivers/video/rockchip/rk_fb.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/rk_fb.c b/drivers/video/rockchip/rk_fb.c
index 05afa24c92b7..5ed678bf8c5e 100644
--- a/drivers/video/rockchip/rk_fb.c
+++ b/drivers/video/rockchip/rk_fb.c
@@ -456,7 +456,7 @@ static int rk_fb_set_par(struct fb_info *info)
 			cblen = crlen = (xvir*yvir);
 			break;
 		default:
-			printk("un supported format:0x%x\n",data_format);
+			printk("%s:un supported format:0x%x\n",__func__,data_format);
 		    return -EINVAL;
 	}
 #else
@@ -498,7 +498,7 @@ static int rk_fb_set_par(struct fb_info *info)
 					par2->yact = par->yact;
 					par2->format = par->format;
 					info2->var.nonstd &= 0xffffff00;
-					info2->var.nonstd |= par->format;
+					info2->var.nonstd |= data_format;
 					dev_drv1->set_par(dev_drv1,layer_id);
 				}
 			}
-- 
2.35.3

